src/blocks/ImageTiles/ImageTiles.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import Ublock from '@src/components/Ublock/Ublock'
+
+export interface ImageTilesProps {
+    uBlockFull: boolean
```

Comment:
It's not clear in the docs, but Image Titles are always a full width block. 
Please remove this option.

Reply:
Done, removed the parameter and set the full screen always true